account_events.go

```diff
@@ -55,101 +55,132 @@ type EventAction string
-	ActionLKENodeCreate            EventAction = "lke_node_create"
-	ActionLongviewClientCreate     EventAction = "longviewclient_create"
-	ActionLongviewClientDelete     EventAction = "longviewclient_delete"
-	ActionLongviewClientUpdate     EventAction = "longviewclient_update"
-	ActionManagedDisabled          EventAction = "managed_disabled"
-	ActionManagedEnabled           EventAction = "managed_enabled"
-	ActionManagedServiceCreate     EventAction = "managed_service_create"
-	ActionManagedServiceDelete     EventAction = "managed_service_delete"
-	ActionNodebalancerCreate       EventAction = "nodebalancer_create"
-	ActionNodebalancerDelete       EventAction = "nodebalancer_delete"
-	ActionNodebalancerUpdate       EventAction = "nodebalancer_update"
-	ActionNodebalancerConfigCreate EventAction = "nodebalancer_config_create"
-	ActionNodebalancerConfigDelete EventAction = "nodebalancer_config_delete"
-	ActionNodebalancerConfigUpdate EventAction = "nodebalancer_config_update"
-	ActionPasswordReset            EventAction = "password_reset"
-	ActionPaymentSubmitted         EventAction = "payment_submitted"
-	ActionStackScriptCreate        EventAction = "stackscript_create"
-	ActionStackScriptDelete        EventAction = "stackscript_delete"
-	ActionStackScriptUpdate        EventAction = "stackscript_update"
-	ActionStackScriptPublicize     EventAction = "stackscript_publicize"
-	ActionStackScriptRevise        EventAction = "stackscript_revise"
-	ActionTFADisabled              EventAction = "tfa_disabled"
-	ActionTFAEnabled               EventAction = "tfa_enabled"
-	ActionTicketAttachmentUpload   EventAction = "ticket_attachment_upload"
-	ActionTicketCreate             EventAction = "ticket_create"
-	ActionTicketUpdate             EventAction = "ticket_update"
-	ActionVolumeAttach             EventAction = "volume_attach"
-	ActionVolumeClone              EventAction = "volume_clone"
-	ActionVolumeCreate             EventAction = "volume_create"
-	ActionVolumeDelte              EventAction = "volume_delete"
-	ActionVolumeUpdate             EventAction = "volume_update"
-	ActionVolumeDetach             EventAction = "volume_detach"
-	ActionVolumeResize             EventAction = "volume_resize"
+	ActionAccountUpdate                 EventAction = "account_update"
+	ActionAccountSettingsUpdate         EventAction = "account_settings_update"
+	ActionBackupsEnable                 EventAction = "backups_enable"
+	ActionBackupsCancel                 EventAction = "backups_cancel"
+	ActionBackupsRestore                EventAction = "backups_restore"
+	ActionCommunityQuestionReply        EventAction = "community_question_reply"
+	ActionCommunityLike                 EventAction = "community_like"
+	ActionCreateCardUpdated             EventAction = "credit_card_updated"
+	ActionDatabaseCreate                EventAction = "database_create"
+	ActionDatabaseDegraded              EventAction = "database_degraded"
```

Comment:
While we're updating Actions, it looks like these database_method actions aren't listed in the API for Event View?

Reply:
Yes, they are missing in the doc, but they exist in this repo before this PR, lol. Let's ask the doc team to add them.